server/main.go

```diff
@@ -41,7 +41,8 @@ func main() {
 
 	log.Info().Str("version", util.Version).Msgf("Starting server")
 
-	kvStore, err := kv.NewKeyValueStore(&config.DefaultConfig.FoundationDB)
+	//kvStore, err := kv.NewKeyValueStore(&config.DefaultConfig.FoundationDB)
```

Comment:
Remove the comment in the final version.

Reply:
done